README.md

```diff
@@ -32,12 +32,6 @@ app.use artsyPassport
   # ----------------
   FACEBOOK_ID: # Facebook app ID
   FACEBOOK_SECRET: # Facebook app secret
-  TWITTER_KEY: # Twitter consumer key
-  TWITTER_SECRET: # Twitter consumer secret
-  TWITTER_KEY: # Twitter consumer key
-  TWITTER_SECRET: # Twitter consumer secret
```

Comment:
Were these duplicated in error?

Reply:
Probably, but we won't need any of them anymore